src/client/js/modules/address-ajax.js

```diff
@@ -0,0 +1,132 @@
+const $ = require('jquery');
+
+class AddressAjax {
+  init(){
+    this.housenumber = $('#input-housenumber');
+    this.postcode = $('#input-postcode');
+    this.csrf = $('input[name=\'csrf\']').val();
+    this.addressButton = $('#addressbutton');
+    this.endpoint = '/address';
+    this.finalEndpoint = '/addresspost';
+    this.formFields = $('.form-fields');
+    this.addressButton.on('click', this.formHandler.bind(this));
+    this.resultListContainerElem = $('.address-results');
+    this.resultListContainerElem.on('click', '.select-link', this.resultItemClickHandler.bind(this));
+    this.addressContinue = $('#addresscontinue');
+    this.confirmContainer = $('.address-confirm');
+    this.confirmContainer.hide();
```

Comment:
@mtamas5 Would be good if you hide the elements with CSS instead of this initialize method. You can add `style="display:none"` to element then you can use `this.confirmContainer.show()`.